models/comment.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+const { selectArticleById } = require("./article.models");
+
+exports.insertComment = (article_id, username, body) => {
+  return selectArticleById(article_id).then((articleExists) => {
```

Comment:
article_id is also a foreign key in the comments table, so a psql error will be thrown if it doesn't exist in the comments table